201-2-vms-loadbalancer-natrules/azuredeploy.json

```diff
@@ -223,16 +160,16 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
+      "apiVersion": "2016-09-01", 
       "type": "Microsoft.Network/loadBalancers/inboundNatRules",
```

Comment:
Latest schema for Microsoft.Network (apiVersion 2016-03-30) doesn't support an inboundNatRules resource type - just the loadBalancers property as Dan mentioned. Be sure to view these templates in VSCode or VS to make sure there are no schema validation warnings